Source/textdat.cpp

```diff
@@ -4,495 +4,496 @@
+	{ N_("The Gateway of Blood and the Halls of Fire are landmarks of mystic origin. Wherever this book you read from resides it is surely a place of great power.\n \nLegends speak of a pedestal that is carved from obsidian stone and has a pool of boiling blood atop its bone encrusted surface. There are also allusions to Stones of Blood that will open a door that guards an ancient treasure...\n \nThe nature of this treasure is shrouded in speculation, my friend, but it is said that the ancient hero Arkaine placed the holy armor Valor in a secret vault. Arkaine was the first mortal to turn the tide of the Sin War and chase the legions of darkness back to the Burning Hells.\n \nJust before Arkaine died, his armor was hidden away in a secret vault. It is said that when this holy armor is again needed, a hero will arise to don Valor once more. Perhaps you are that hero... |"),
 	    true, TSFX_STORY15 },
-	{ "Every child hears the story of the warrior Arkaine and his mystic armor known as Valor. If you could find its resting place, you would be well protected against the evil in the Labyrinth. |",
+	{ N_("Every child hears the story of the warrior Arkaine and his mystic armor known as Valor. If you could find its resting place, you would be well protected against the evil in the Labyrinth. |"),
 	    true, TSFX_TAVERN13 },
-	{ "Hmm... it sounds like something I should remember, but I've been so busy learning new cures and creating better elixirs that I must have forgotten. Sorry... |",
+	{ N_("Hmm... it sounds like something I should remember, but I've been so busy learning new cures and creating better elixirs that I must have forgotten. Sorry... |"),
 	    true, TSFX_HEALER13 },
-	{ "The story of the magic armor called Valor is something I often heard the boys talk about. You had better ask one of the men in the village. |",
+	{ N_("The story of the magic armor called Valor is something I often heard the boys talk about. You had better ask one of the men in the village. |"),
 	    true, TSFX_BMAID13 },
-	{ "The armor known as Valor could be what tips the scales in your favor. I will tell you that many have looked for it - including myself. Arkaine hid it well, my friend, and it will take more than a bit of luck to unlock the secrets that have kept it concealed oh, lo these many years. |",
+	{ N_("The armor known as Valor could be what tips the scales in your favor. I will tell you that many have looked for it - including myself. Arkaine hid it well, my friend, and it will take more than a bit of luck to unlock the secrets that have kept it concealed oh, lo these many years. |"),
 	    true, TSFX_SMITH14 },
-	{ "Zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz... |", true, TSFX_DRUNK15 },
-	{ "Should you find these Stones of Blood, use them carefully. \n \nThe way is fraught with danger and your only hope rests within your self trust. |",
+	{ N_("Zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz... |"), true, TSFX_DRUNK15 },

```

Comment:
@qndel will go "Chrrrrrrrrr" when he sees this!

Reply:
This was actually translated in the Swedish PS1 PAL version (same string value though) :D

Reply:
Yeah, not pointing it out as a mistake, just making an internal joke :)

Reply:
:)